api/cloud/oc_cloud_manager.c

```diff
@@ -444,11 +475,15 @@ cloud_manager_register_async(void *data)
   return OC_EVENT_DONE;
 
 retry:
+  if (!cloud_schedule_retry(ctx, OC_CLOUD_ACTION_REGISTER,
+                            cloud_manager_register_async,
+                            OC_STATUS_SERVICE_UNAVAILABLE)) {
```

Comment:
Why `OC_STATUS_SERVICE_UNAVAILABLE`, seems like internal error for both code branches that jump here

Reply:
ok removed 3 parameter.